src/containers/shared/components/CopyToClipboard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import copy from '../images/copy.png'
+
+interface Props {
+  text: string
+  className?: string
+}
+
+const CopyToClipboard = ({ text, className }: Props) => (
+  <input
+    className={className}
+    type="image"
+    src={copy}
+    alt="Copy"
+    onClick={() => navigator.clipboard.writeText(text)}
+  />
+)
+CopyToClipboard.propTypes = {
+  text: PropTypes.string,
+  className: PropTypes.string,
+}
+
+CopyToClipboard.defaultProps = {
+  text: '',
+  className: null,
+}
+
+export default CopyToClipboard
```

Comment:
Default exports are not preferred.

Reply:
huh, isnt it the convention throughout the repo to use this? or are we moving away from it?